### DISCUSSION TITLE
Does maddy come with a webmail?

### DISCUSSION THREAD

**Discussion by hiqsociety** [emersion/maddy#500] [Q&A]
1. Does maddy come with a webmail?
2. Spam protection features? how does it work?

**Comment by foxcpp** [comment]
>1. Does maddy come with a webmail?

No. You can use [alps](http://sr.ht/~migadu/alps) or some PHP-based webmail, depending on your needs.

>2.Spam protection features? how does it work?

maddy has very basic built-in spam protection - it can check sender addresses against DNSBLs. It is not active by default, though.

If you are in need of a more complex spam filtering solution, rspamd is recommended. maddy supports rspamd's HTTP protocol via check.rspamd module.

**Reply by kolinfluence** [reply]
this doesnt seemed maintained for 2 years

https://sr.ht/~emersion/alps/

any golang webmail equivalent or better?

**Reply by foxcpp** [reply]
migadu fork is maintained with latest commit 2 days ago: https://sr.ht/~migadu/alps/
